.github/workflows/trunk.yml

```diff
@@ -0,0 +1,20 @@
+name: Trunk Workflow
+on:
+    push:
+        branches:
+            - trunk
+            - add/run-e2e-tests-on-trunk
```

Comment:
I think it will be removed before merging.

Reply:
Yes, it is only for our tests

Reply:
Remove here 8f728ff1df60fdae32ef90cf55994ad75afad7e4